Proiect/new-evaluation.css

```diff
@@ -67,5 +80,64 @@
 		}
 
 		li {
-			display: inline;
+			float: left;
+			text-align: left;
+			width: 16%;
+			margin: 2px;
+			min-height: 36px;
+			font-family: Arial;
+			font-size: 15px;
+		}
+
+		#submit {
+			float: right;
+			width: 150px;
+			height: 50px;
+			background-color: #419fe2;
+			color: white;
+			border: none;
+			border-radius: 2px;
+			font-size: 20px;
+			box-shadow: 0px 3px 4px grey;
+		}
+
+		#div_footer {
```

Comment:
I was having some difficulties while trying to fit the footer into the page, so I tried making it a div instead. But I fixed it now and it works just fine with a <footer>.